test/integration/controller_test.go

```diff
@@ -1333,7 +1565,7 @@ func newTestController(t *testing.T, config fakeosb.FakeClientConfiguration) (
 	informerFactory := scinformers.NewSharedInformerFactory(catalogClient, 10*time.Second)
 	serviceCatalogSharedInformers := informerFactory.Servicecatalog().V1beta1()
 
-	fakeRecorder := record.NewFakeRecorder(10)
+	fakeRecorder := record.NewFakeRecorder(20)
```

Comment:
can you put a comment here on why this was necessary so if people in the future get stuck they might know what to do?
Also, why not something larger like 100?

Reply:
Done. And I changed it to 50, which I expect should never be hit by our tests.